docs/content/topics/developers.md

```diff
@@ -60,7 +60,7 @@ $ git clone https://github.com/brigadecore/brigade $GOPATH/src/github.com/brigad
 $ cd $GOPATH/src/github.com/brigadecore/brigade
 ```
 
-**Note**: this leaves you at the tip of **master** in the repository where active development
+**Note**: this leaves you at the tip of **main** in the repository where active development
```

Comment:
Do we want to get even more explicit here and mention `where active development on Brigade v2.x is happening`?  And/or offer the reminder that the tip of v1 development (even though about to be in maintenance mode) can be checked out via the `v1` branch.

Reply:
I started out on that path, initially, and then second guessed it. My worry was that I'd rather not promote unsolicited investment in v1. Any thoughts?

Reply:
sgtm 👍